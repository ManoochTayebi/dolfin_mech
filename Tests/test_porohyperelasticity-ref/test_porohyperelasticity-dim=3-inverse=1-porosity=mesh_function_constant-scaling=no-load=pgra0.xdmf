<?xml version="1.0"?>
<!DOCTYPE Xdmf SYSTEM "Xdmf.dtd" []>
<Xdmf Version="3.0" xmlns:xi="http://www.w3.org/2001/XInclude">
  <Domain>
    <Grid Name="TimeSeries" GridType="Collection" CollectionType="Temporal">
      <Grid Name="mesh" GridType="Uniform">
        <Topology NumberOfElements="6000" TopologyType="Tetrahedron" NodesPerElement="4">
          <DataItem Dimensions="6000 4" NumberType="UInt" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/Mesh/0/mesh/topology</DataItem>
        </Topology>
        <Geometry GeometryType="XYZ">
          <DataItem Dimensions="1331 3" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/Mesh/0/mesh/geometry</DataItem>
        </Geometry>
        <Time Value="0" />
        <Attribute Name="u" AttributeType="Vector" Center="Node">
          <DataItem Dimensions="1331 3" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/0</DataItem>
        </Attribute>
        <Attribute Name="phis0" AttributeType="Scalar" Center="Cell">
          <DataItem Dimensions="6000 1" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/1</DataItem>
        </Attribute>
        <Attribute Name="p_surface" AttributeType="Scalar" Center="Node">
          <DataItem Dimensions="1331 1" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/2</DataItem>
        </Attribute>
        <Attribute Name="gamma" AttributeType="Scalar" Center="Cell">
          <DataItem Dimensions="6000 1" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/3</DataItem>
        </Attribute>
        <Attribute Name="lbda" AttributeType="Vector" Center="Cell">
          <DataItem Dimensions="6000 3" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/4</DataItem>
        </Attribute>
        <Attribute Name="mu" AttributeType="Vector" Center="Cell">
          <DataItem Dimensions="6000 3" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/5</DataItem>
        </Attribute>
        <Attribute Name="u_old" AttributeType="Vector" Center="Node">
          <DataItem Dimensions="1331 3" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/6</DataItem>
        </Attribute>
        <Attribute Name="phis0_old" AttributeType="Scalar" Center="Cell">
          <DataItem Dimensions="6000 1" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/7</DataItem>
        </Attribute>
        <Attribute Name="p_surface_old" AttributeType="Scalar" Center="Node">
          <DataItem Dimensions="1331 1" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/8</DataItem>
        </Attribute>
        <Attribute Name="gamma_old" AttributeType="Scalar" Center="Cell">
          <DataItem Dimensions="6000 1" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/9</DataItem>
        </Attribute>
        <Attribute Name="lbda_old" AttributeType="Vector" Center="Cell">
          <DataItem Dimensions="6000 3" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/10</DataItem>
        </Attribute>
        <Attribute Name="mu_old" AttributeType="Vector" Center="Cell">
          <DataItem Dimensions="6000 3" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/11</DataItem>
        </Attribute>
        <Attribute Name="F" AttributeType="Tensor" Center="Cell">
          <DataItem Dimensions="6000 9" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/12</DataItem>
        </Attribute>
        <Attribute Name="J" AttributeType="Scalar" Center="Cell">
          <DataItem Dimensions="6000 1" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/13</DataItem>
        </Attribute>
        <Attribute Name="C" AttributeType="Tensor" Center="Cell">
          <DataItem Dimensions="6000 9" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/14</DataItem>
        </Attribute>
        <Attribute Name="E" AttributeType="Tensor" Center="Cell">
          <DataItem Dimensions="6000 9" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/15</DataItem>
        </Attribute>
        <Attribute Name="phis" AttributeType="Scalar" Center="Cell">
          <DataItem Dimensions="6000 1" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/16</DataItem>
        </Attribute>
        <Attribute Name="phif" AttributeType="Scalar" Center="Cell">
          <DataItem Dimensions="6000 1" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/17</DataItem>
        </Attribute>
        <Attribute Name="phif0" AttributeType="Scalar" Center="Cell">
          <DataItem Dimensions="6000 1" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/18</DataItem>
        </Attribute>
        <Attribute Name="Phis0" AttributeType="Scalar" Center="Cell">
          <DataItem Dimensions="6000 1" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/19</DataItem>
        </Attribute>
        <Attribute Name="Phif0" AttributeType="Scalar" Center="Cell">
          <DataItem Dimensions="6000 1" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/20</DataItem>
        </Attribute>
        <Attribute Name="Sigma_skel" AttributeType="Tensor" Center="Cell">
          <DataItem Dimensions="6000 9" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/21</DataItem>
        </Attribute>
        <Attribute Name="sigma_skel" AttributeType="Tensor" Center="Cell">
          <DataItem Dimensions="6000 9" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/22</DataItem>
        </Attribute>
        <Attribute Name="dWbulkdPhis" AttributeType="Scalar" Center="Cell">
          <DataItem Dimensions="6000 1" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/23</DataItem>
        </Attribute>
        <Attribute Name="Sigma_bulk" AttributeType="Tensor" Center="Cell">
          <DataItem Dimensions="6000 9" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/24</DataItem>
        </Attribute>
        <Attribute Name="sigma_bulk" AttributeType="Tensor" Center="Cell">
          <DataItem Dimensions="6000 9" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/25</DataItem>
        </Attribute>
        <Attribute Name="pf" AttributeType="Scalar" Center="Cell">
          <DataItem Dimensions="6000 1" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/26</DataItem>
        </Attribute>
      </Grid>
      <Grid Name="mesh" GridType="Uniform">
        <Topology NumberOfElements="6000" TopologyType="Tetrahedron" NodesPerElement="4">
          <DataItem Dimensions="6000 4" NumberType="UInt" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/Mesh/27/mesh/topology</DataItem>
        </Topology>
        <Geometry GeometryType="XYZ">
          <DataItem Dimensions="1331 3" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/Mesh/27/mesh/geometry</DataItem>
        </Geometry>
        <Time Value="1" />
        <Attribute Name="u" AttributeType="Vector" Center="Node">
          <DataItem Dimensions="1331 3" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/27</DataItem>
        </Attribute>
        <Attribute Name="phis0" AttributeType="Scalar" Center="Cell">
          <DataItem Dimensions="6000 1" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/28</DataItem>
        </Attribute>
        <Attribute Name="p_surface" AttributeType="Scalar" Center="Node">
          <DataItem Dimensions="1331 1" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/29</DataItem>
        </Attribute>
        <Attribute Name="gamma" AttributeType="Scalar" Center="Cell">
          <DataItem Dimensions="6000 1" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/30</DataItem>
        </Attribute>
        <Attribute Name="lbda" AttributeType="Vector" Center="Cell">
          <DataItem Dimensions="6000 3" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/31</DataItem>
        </Attribute>
        <Attribute Name="mu" AttributeType="Vector" Center="Cell">
          <DataItem Dimensions="6000 3" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/32</DataItem>
        </Attribute>
        <Attribute Name="u_old" AttributeType="Vector" Center="Node">
          <DataItem Dimensions="1331 3" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/33</DataItem>
        </Attribute>
        <Attribute Name="phis0_old" AttributeType="Scalar" Center="Cell">
          <DataItem Dimensions="6000 1" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/34</DataItem>
        </Attribute>
        <Attribute Name="p_surface_old" AttributeType="Scalar" Center="Node">
          <DataItem Dimensions="1331 1" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/35</DataItem>
        </Attribute>
        <Attribute Name="gamma_old" AttributeType="Scalar" Center="Cell">
          <DataItem Dimensions="6000 1" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/36</DataItem>
        </Attribute>
        <Attribute Name="lbda_old" AttributeType="Vector" Center="Cell">
          <DataItem Dimensions="6000 3" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/37</DataItem>
        </Attribute>
        <Attribute Name="mu_old" AttributeType="Vector" Center="Cell">
          <DataItem Dimensions="6000 3" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/38</DataItem>
        </Attribute>
        <Attribute Name="F" AttributeType="Tensor" Center="Cell">
          <DataItem Dimensions="6000 9" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/39</DataItem>
        </Attribute>
        <Attribute Name="J" AttributeType="Scalar" Center="Cell">
          <DataItem Dimensions="6000 1" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/40</DataItem>
        </Attribute>
        <Attribute Name="C" AttributeType="Tensor" Center="Cell">
          <DataItem Dimensions="6000 9" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/41</DataItem>
        </Attribute>
        <Attribute Name="E" AttributeType="Tensor" Center="Cell">
          <DataItem Dimensions="6000 9" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/42</DataItem>
        </Attribute>
        <Attribute Name="phis" AttributeType="Scalar" Center="Cell">
          <DataItem Dimensions="6000 1" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/43</DataItem>
        </Attribute>
        <Attribute Name="phif" AttributeType="Scalar" Center="Cell">
          <DataItem Dimensions="6000 1" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/44</DataItem>
        </Attribute>
        <Attribute Name="phif0" AttributeType="Scalar" Center="Cell">
          <DataItem Dimensions="6000 1" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/45</DataItem>
        </Attribute>
        <Attribute Name="Phis0" AttributeType="Scalar" Center="Cell">
          <DataItem Dimensions="6000 1" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/46</DataItem>
        </Attribute>
        <Attribute Name="Phif0" AttributeType="Scalar" Center="Cell">
          <DataItem Dimensions="6000 1" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/47</DataItem>
        </Attribute>
        <Attribute Name="Sigma_skel" AttributeType="Tensor" Center="Cell">
          <DataItem Dimensions="6000 9" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/48</DataItem>
        </Attribute>
        <Attribute Name="sigma_skel" AttributeType="Tensor" Center="Cell">
          <DataItem Dimensions="6000 9" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/49</DataItem>
        </Attribute>
        <Attribute Name="dWbulkdPhis" AttributeType="Scalar" Center="Cell">
          <DataItem Dimensions="6000 1" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/50</DataItem>
        </Attribute>
        <Attribute Name="Sigma_bulk" AttributeType="Tensor" Center="Cell">
          <DataItem Dimensions="6000 9" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/51</DataItem>
        </Attribute>
        <Attribute Name="sigma_bulk" AttributeType="Tensor" Center="Cell">
          <DataItem Dimensions="6000 9" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/52</DataItem>
        </Attribute>
        <Attribute Name="pf" AttributeType="Scalar" Center="Cell">
          <DataItem Dimensions="6000 1" Format="HDF">test_porohyperelasticity-dim=3-inverse=1-porosity=mesh_function_constant-scaling=no-load=pgra0.h5:/VisualisationVector/53</DataItem>
        </Attribute>
      </Grid>
    </Grid>
  </Domain>
</Xdmf>
