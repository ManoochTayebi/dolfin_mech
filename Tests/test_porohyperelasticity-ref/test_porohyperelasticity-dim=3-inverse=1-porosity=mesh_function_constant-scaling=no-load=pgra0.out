k_step = 1
 |  k_t = 1
 |   |  dt = 1.0
 |   |  t = 1.0
 |   |  t_step = 1.0
 |   |  k_iter = 1
 |   |   |  Assembly… 24.50905990600586 s
 |   |   |  res_norm      = 3.0000e+00
 |   |   |  Solve… 0.3430299758911133 s
 |   |   |  du_norm       = 2.2924e+00
 |   |   |  dphis0_norm   = 0.0000e+00
 |   |   |  dp_surface_norm = 1.2747e+01
 |   |   |  dgamma_norm   = 1.3933e-10
 |   |   |  dlbda_norm    = 4.6339e-02
 |   |   |  dmu_norm      = 7.6954e-14
 |   |   |  u_norm        = 2.2924e+00
 |   |   |  u_norm_old    = 0.0000e+00
 |   |   |  phis0_norm    = 3.8730e+01
 |   |   |  phis0_norm_old = 3.8730e+01
 |   |   |  p_surface_norm = 1.2747e+01
 |   |   |  p_surface_norm_old = 0.0000e+00
 |   |   |  gamma_norm    = 1.3933e-10
 |   |   |  gamma_norm_old = 0.0000e+00
 |   |   |  lbda_norm     = 4.6339e-02
 |   |   |  lbda_norm_old = 0.0000e+00
 |   |   |  mu_norm       = 7.6954e-14
 |   |   |  mu_norm_old   = 0.0000e+00
 |   |   |  u_err         = 1.0000e+00
 |   |   |  phis0_err     = 0.0000e+00
 |   |   |  p_surface_err = 1.0000e+00
 |   |   |  gamma_err     = 0.0000e+00
 |   |   |  lbda_err      = 1.0000e+00
 |   |   |  mu_err        = 0.0000e+00
 |   |  k_iter = 2
 |   |   |  Assembly… 0.08165192604064941 s
 |   |   |  res_norm      = 1.7919e-03
 |   |   |  res_err       = 5.9728e-04
 |   |   |  dres_norm     = 3.0000e+00
 |   |   |  res_err_rel   = 1.0000e+00
 |   |   |  Solve… 0.2869541645050049 s
 |   |   |  du_norm       = 5.1354e-02
 |   |   |  dphis0_norm   = 6.4935e-06
 |   |   |  dp_surface_norm = 5.0490e-04
 |   |   |  dgamma_norm   = 4.9163e-15
 |   |   |  dlbda_norm    = 2.6410e-14
 |   |   |  dmu_norm      = 9.5480e-15
 |   |   |  relax         = 1.0000e+00
 |   |   |  u_norm        = 2.3438e+00
 |   |   |  u_norm_old    = 0.0000e+00
 |   |   |  phis0_norm    = 3.8730e+01
 |   |   |  phis0_norm_old = 3.8730e+01
 |   |   |  p_surface_norm = 1.2747e+01
 |   |   |  p_surface_norm_old = 0.0000e+00
 |   |   |  gamma_norm    = 1.3932e-10
 |   |   |  gamma_norm_old = 0.0000e+00
 |   |   |  lbda_norm     = 4.6339e-02
 |   |   |  lbda_norm_old = 0.0000e+00
 |   |   |  mu_norm       = 8.5655e-14
 |   |   |  mu_norm_old   = 0.0000e+00
 |   |   |  u_err         = 2.1911e-02
 |   |   |  phis0_err     = 1.6766e-07
 |   |   |  p_surface_err = 3.9610e-05
 |   |   |  gamma_err     = 0.0000e+00
 |   |   |  lbda_err      = 5.6994e-13
 |   |   |  mu_err        = 0.0000e+00
 |   |  k_iter = 3
 |   |   |  Assembly… 0.07981586456298828 s
 |   |   |  res_norm      = 4.3217e-08
 |   |   |  res_err       = 1.4406e-08
 |   |   |  dres_norm     = 1.7918e-03
 |   |   |  res_err_rel   = 9.9998e-01
 |   |   |  Solve… 0.29818105697631836 s
 |   |   |  du_norm       = 1.0430e-06
 |   |   |  dphis0_norm   = 3.2387e-08
 |   |   |  dp_surface_norm = 3.8358e-07
 |   |   |  dgamma_norm   = 2.6746e-19
 |   |   |  dlbda_norm    = 2.4299e-16
 |   |   |  dmu_norm      = 1.9365e-15
 |   |   |  relax         = 1.0000e+00
 |   |   |  u_norm        = 2.3438e+00
 |   |   |  u_norm_old    = 0.0000e+00
 |   |   |  phis0_norm    = 3.8730e+01
 |   |   |  phis0_norm_old = 3.8730e+01
 |   |   |  p_surface_norm = 1.2747e+01
 |   |   |  p_surface_norm_old = 0.0000e+00
 |   |   |  gamma_norm    = 1.3932e-10
 |   |   |  gamma_norm_old = 0.0000e+00
 |   |   |  lbda_norm     = 4.6339e-02
 |   |   |  lbda_norm_old = 0.0000e+00
 |   |   |  mu_norm       = 8.6149e-14
 |   |   |  mu_norm_old   = 0.0000e+00
 |   |   |  u_err         = 4.4503e-07
 |   |   |  phis0_err     = 8.3624e-10
 |   |   |  p_surface_err = 3.0092e-08
 |   |   |  gamma_err     = 0.0000e+00
 |   |   |  lbda_err      = 5.2437e-15
 |   |   |  mu_err        = 0.0000e+00
 |   |   |  Nonlinear solver converged…
